docs/policies_and_procedures/cruise_documentation.md

```diff
@@ -1,3 +1,11 @@
 # Cruise Documentation
 
-**Cruise Reports** are the primary cruise documentation and are provided on the cruise page when available. The CCHDO attempts to make other relevant documentation, especially if received in electronic form, available as well. This includes reports from investigators, overviews from data quality examination, copies of tables of information held by the CCHDO, etc.
+**Cruise Reports** are the primary cruise documentation and are provided on the cruise page when available.
+
+The CCHDO attempts to make other relevant documentation, especially if received in electronic form, available as well.
+This includes reports from investigators, overviews from data quality examination, and copies of tables of information held by the CCHDO.
+The appendix of the detailed submission guide (and an abridged version in the summary doc) has recommendations for cruise report content.
+
+See [Appendix A - Outline of a Cruise Report](../submitting_data/detailed_guide.md#appendix-a-outline-of-a-cruise-report).
```

Comment:
For some reason this links as expected in the built ver, but emits a warning on build. I recommend it be changed to `See <project:#outline_cruise_report>`. The reason for this is because there is an explicit link target for this header that is resolvable globally at build time.